newsrevenuehub/rev-engine
# #9579 New pages start with default content

#### What's this PR do?
New pages are created with default content, including an admin configurable default header logo.

#### How should this be manually tested?
Create a new page and observe the default settings.
If you want to test the default header logo as well, go in to django admin and add a "logo" to the DefaultHeaderLogo model and save it. Then create a new page and notice that this is used as the header logo.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No, aside from setting the default header logo
